pkg/flare/sysprobe/archive_nolinux.go

```diff
@@ -18,3 +18,8 @@ func GetSystemProbeConntrackCached(_ string) ([]byte, error) {
 func GetSystemProbeConntrackHost(_ string) ([]byte, error) {
 	return nil, errors.New("GetSystemProbeConntrackHost is not supported without the process agent on linux")
 }
+
+// GetSystemProbeBTFLoaderInfo is not supported without the process agent on linux
+func GetSystemProbeBTFLoaderInfo(_ string) ([]byte, error) {
+	return nil, errors.New("GetSystemProbeBTFLoaderInfo is not supported without the process agent on linux")
```

Comment:
Why is this true? Querying the system-probe endpoint does not require process-agent in any way.

Reply:
The comment is misleading though, because the core agent can and does query the system-probe directly using the `RemoteSysProbeUtil`. Only NPM goes through the process-agent.

Reply:
That's a fair point, will update

Reply:
The error still mentions process agent. Maybe simplify and say `GetSystemProbeBTFLoaderInfo is not supported`.

Reply:
Updated this